merlin-framework-processor/src/main/java/gov/nasa/jpl/aerie/merlin/processor/MissionModelParser.java

```diff
@@ -260,6 +264,37 @@ private ActivityTypeRecord parseActivityType(final PackageElement missionModelEl
     return new ActivityTypeRecord(name, activityTypeElement, parameters, validations, mapper, defaultsStyle, effectModel);
   }
 
+  private static void validateControllableDurationParameter(
+      final String activityName,
+      final List<ParameterRecord> parameters,
+      final String durationParameterName)
+  throws InvalidMissionModelException
+  {
+    final var durationParameterRecord = lookupParameterByName(parameters, durationParameterName);
+    if (durationParameterRecord.isEmpty()) {
+      throw new InvalidMissionModelException(
+          "In activity " + activityName + ", \"" + durationParameterName + "\"" +
+          " is declared as the ControllableDuration parameter, but there is no parameter with that name");
+    }
+    final var record = durationParameterRecord.get();
+    if (!record.type.toString().equals("gov.nasa.jpl.aerie.merlin.protocol.types.Duration")) {
```

Comment:
Can't you import the Duration class and then compare to `Duration.getClass().getName()` or something like that? As is, if we ever rearrange our package structure here we'll have to change this hardcoded line.

Reply:
I ended up with this:

```java
this.typeUtils.isSameType(record.type, this.elementUtils.getTypeElement(Duration.class.getName()).asType())
```

Verbose, but robust to moving/renaming `Duration`.